fitsnap3/io/sections/scraper.py

```diff
@@ -8,4 +8,7 @@ def __init__(self, name, config, args):
         self.scraper = self.get_value("SCRAPER", "scraper", "JSON")
         self.save_group_scrape = self.get_value("SCRAPER", "save_group_scrape", "None", "str")
         self.read_group_scrape = self.get_value("SCRAPER", "read_group_scrape", "None", "str")
+        self.property_array = self.get_value("SCRAPER", "property_array", "None", "str")
+        self.unit_array = self.get_value("SCRAPER", "unit_array", "None", "str")
+        self.unit_system = self.get_value("SCRAPER", "unit_system", "None", "str")
```

Comment:
> Is it worth adding (yet) another section to the input file just in case people want to define their own conversion style?

@mitwood This is how I was planning on adding user units. There will be one for the scraper and one for the calculator.